### DISCUSSION TITLE
How do I setup timeout?

### DISCUSSION THREAD

**Discussion by malkoG** [coollabsio/coolify#296] [Feature Requests]
Thanksfully, I am using coolify well for building small services. 
But, some services do heavy job and they have to respond to other server's request.

Because of timeout, other server can't receive desired response.

How can I customize service's timeout?

**Comment by andrasbacsai** [comment]
Do you mean the timeout for each HTTP request? That is defined in the haproxy configuration. Since you asked the question, I've modified the timeout values to 120 from 60. 

Do you need a custom timeout value for each application (or globally to all applications)? 

I can make a setting where you can define custom timeouts on the UI.

**Comment by markmals** [comment]
Hello, I am also in need of a setting to manually configure the timeout per app, as the current one is not long enough. I don't see a place to do this currently. Is this still on the roadmap? @andrasbacsai

**Reply by andrasbacsai** [reply]
With Traefik, you won't have this problem.
